packages/atlas/queryFilter.json

```diff
@@ -0,0 +1,7 @@
+{
+  "video": [],
+  "dataObject": {
+    "video": [],
+    "thumbnail": []
+  }
+}
```

Comment:
It's not very clear what you need to pass here. At least to me. Maybe we could rename this file `blacklists.json` and in this file use `videoIds`, `videoDataObjectIds`, `thumbnailDataObjectIds`?